frontend/src/components/layout/base/BaseHeader.vue

```diff
@@ -1,31 +1,10 @@
 <script setup lang="ts">
```

Comment:
why is all this code removed

Reply:
It hasn't been removed, it has been relocated.